src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
```

Comment:
Consider importing React at the top of the file for better readability and consistency. 📚